### PR TITLE
[W5.7][W13-1]Auyok Sean

### PR DESCRIPTION
**Pull request by A19Sean**
Add sorting functionality to address book.

### CODE REVIEW

**Comment by AyushChatto** [comment]
Good job on the test cases. Very extensive and accounts for edge cases. Consider not importing the individual commands in the parser, instead of the whole library (I understand IntelliJ does this automatically, so you might have to go back and fix).

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (asiddharth) [inline src/seedu/addressbook/parser/Parser.java:14]: Minor comment.
Imported classes should always be listed explicitly.

Please look at https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements

File: src/seedu/addressbook/commands/SortCommand.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+/**
```
Comment (asiddharth) [inline src/seedu/addressbook/commands/SortCommand.java:7]: Good job on adding the header comment!

**Review by asiddharth — APPROVED** [review]
Kudos @A19Sean ! Good job!
